experimental/framework/tinylicious-client/src/test/ResettableTinyliciousClient.ts

```diff
@@ -1,12 +0,0 @@
-/*!
```

Comment:
did this just not get deleted before?  should ideally delete this in a separate pr for clarity

Reply:
it is deleted from the previous PR and reflected in the FluidFramework main branch. I tried removing it but wasn't able to when I first created the PR 🤔